### PR TITLE
fix: dev: added support for non default schema in hive metadata api and ...

### PR DESCRIPTION
**Pull request by dineshrathi**
schema builder api and hive metadata api fix for schema name support, incorporated review comments, also created fresh branch

### CODE REVIEW

File: src/main/java/com/qubole/qds/sdk/java/details/StoreTablePropertiesBuilderImpl.java [hunk 1]
```diff
@@ -52,12 +53,13 @@ public StoreTablePropertiesBuilder columns(Map<String, String> columns)
     protected InvokeArguments<Status> getInvokeArguments()
     {
         RequestDetails entity = CommandBuilderImplBase.makeJsonEntity(node, RequestDetails.Method.POST);
```
Comment (rjainqb) [inline src/main/java/com/qubole/qds/sdk/java/details/StoreTablePropertiesBuilderImpl.java]: We will need to add function to set the schemaName here just like in SchemaCommandBuilder.

File: src/main/java/com/qubole/qds/sdk/java/details/StoreTablePropertiesBuilderImpl.java [hunk 2]
```diff
@@ -26,6 +26,7 @@
     private final ObjectNode node = QdsClientImpl.getMapper().createObjectNode();
     private final QdsClient client;
     private final String tableName;
+    private final String schemaName;
```
Comment (rjainqb) [inline src/main/java/com/qubole/qds/sdk/java/details/StoreTablePropertiesBuilderImpl.java]: We will need to add function to set the schemaName here just like in SchemaCommandBuilder.
Add a line note

Comment (dineshrathi) [inline src/main/java/com/qubole/qds/sdk/java/details/StoreTablePropertiesBuilderImpl.java]: actually for all functions which work with a table name, we have allowed one to specify <schemaname>.<tablename> syntax, including this one as well.

Comment (rjainqb) [inline src/main/java/com/qubole/qds/sdk/java/details/StoreTablePropertiesBuilderImpl.java]: my bad. I didn't fully read the constructor. In addition to this, can we keep the original constructor too to make it backward compatible?

StorageTablePropertiesBuilderImpl(QdsClient, String tableName) {
  // parse tableName here
}

Comment (dineshrathi) [inline src/main/java/com/qubole/qds/sdk/java/details/StoreTablePropertiesBuilderImpl.java]: This is an impl class, the only usage within the sdk has been fixed, what is expected usage wrt such classes?

**Comment by mindprince** [comment]
Looks good to me (modulo the discussion here https://github.com/qubole/qds-sdk-java/pull/26#discussion_r23117280, but that's mostly preference.)

**Comment by qubole** [comment]
Please merge if its fine, o/w if any further changes are needed, let me
know.

**Pull request merged**